docs/installation_c_plus_plus.md

```diff
@@ -0,0 +1,129 @@
+# C++ Installation
+
+!!! note
+      This version of the documentation is for installation of Xailient SDK using wheel file (.whl). 
+      If you are looking for documentation for installion of Xailient SDK using SDK package, please visit our old [documentation](https://xailient.readthedocs.io/en/latest/installation/).
+      
+      If your local machines doesn't meet the requirements for native installation we also offer Docker [Containers as a deployment option](https://xailient-docs.readthedocs.io/en/latest/container/).
+
+
+## Device Setup
+
+To meet the minimum requirements, your device must be a x86_64 or ARM32 architecture device. We've tested the following operating systems:
+
+* Ubuntu 18.04 LTS
+* Ubuntu 19.04
+* Raspbian Buster
+
+!!! note
+
+    To run Xailient SDK on Windows or MacOS, please run it inside a VirtualBox.
+
+    Here are some resources you can refer to to install Ubuntu operating system on a VirtualBox:
+
+    * [Resource 1](https://brb.nci.nih.gov/seqtools/installUbuntu.html)
+    * [Resource 2](https://medium.com/@tushar0618/install-ubuntu-16-04-lts-on-virtual-box-desktop-version-30dc6f1958d0)
+    * [Resource 3](https://itsfoss.com/install-linux-in-virtualbox/)
+
+
+For ARM32, we have tested on the following device:
+
+* RaspberryPi 3B+
+
+### Installing C++ SDK
+
+!!! note
+
+    If you are installing Python SDK, please refer to this [section](https://xailient-docs.readthedocs.io/en/latest/container/).
```

Comment:
This link refers to ".../latest/container" which I presume is our information on running the SDK inside a container. Shouldn't this point to the docs in installation_python.md instead?